lib/stripe/api_resource.rb

```diff
@@ -2,6 +2,13 @@ module Stripe
   class APIResource < StripeObject
     include Stripe::APIOperations::Request
 
+    # A flag that can be set a behavior that will cause this resource to be
+    # encoded and sent up along with an update of its parent resource. This is
+    # usually not desirable because resources are updated individually on their
+    # own endpoints, but there are certain cases, changing a customer's default
```

Comment:
Yeah, I got what you were saying but in the future if someone was reading it without context it might have required a second pass through or something.

Yeah using customer.source deletes the old default if it exists, creates a new card, and sets the new card as the default